python/lsst/obs/sdss/yanny.py

```diff
@@ -1,61 +1,302 @@
-#
-# yanny.py
-#
-# Python library for reading & writing yanny files.
-#
-# B. A. Weaver, NYU, 2008-10-20
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-# 
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-# 
-# You should have received a copy of the LSST License Statement and 
-# the GNU General Public License along with this program.  If not, 
-# see <http://www.lsstcorp.org/LegalNotices/>.
-#
-"""
-yanny is an object-oriented interface to FTCL/yanny data files following the
-specifications at http://www.sdss.org/dr6/dm/flatFiles/yanny.html.  The format
-of the returned object is similar to that returned by read_yanny() in the
-efftickle perl package (in the photoop product).
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
I think you need to copy the appropriate `LICENSE.rst` into this directory, possibly as `yanny-LICENSE.rst`.